hierarchical_review_plots/polar_bar_plotly.py

```diff
@@ -83,7 +83,7 @@ def _update_bar_polar(fig):
         polar_radialaxis_ticksuffix="",
         polar_angularaxis_rotation=90,
         polar_angularaxis_direction="clockwise",
-        polar_radialaxis_dtick=10,
-        polar_hole=0.10,
+        polar_radialaxis_dtick=20,
+        polar_hole=0.30,
```

Comment:
Tal vez por aquí pueda ser el cotorreo:
```suggestion
        polar_hole=0.30,
        polar = dict(
            bgcolor = "rgb(223, 223, 223)",
            angularaxis = dict(
                linewidth = 3,
                showline=True,
                linecolor='black'
            ),
        )
```
Tal vez cambiar el fondo (regresarlo a blanco) cambiar los colores de las líneas a blanco también.
De [aquí](https://plotly.com/python/polar-chart/#polar-bar-chart) lo saqué. Parece que debería de ser utilizando
`fig = go.Figure(go.Barpolar(`.